docs/building-spark.md

```diff
@@ -79,15 +79,20 @@ Example:
 
     ./build/mvn -Pyarn -Dhadoop.version=2.8.5 -DskipTests clean package
 
+    ./build/mvn -Pyarn -Phadoop-3.2 -Dhadoop.version=3.1.2 -DskipTests clean package
+
 ## Building With Hive and JDBC Support
 
 To enable Hive integration for Spark SQL along with its JDBC server and CLI,
 add the `-Phive` and `Phive-thriftserver` profiles to your existing build options.
-By default Spark will build with Hive 1.2.1 bindings.
+By default, Spark will build hadoop-2.7 with Hive 1.2.1 and hadoop-3.2 with Hive 2.3.5.
```

Comment:
This wordings look a bit weird - I think we build with hadoop-2.7 by default and optionally we can build with hadoop-3.2 and Hive 2.3.5. Not both by default.

Reply:
How about?
`By default, Spark will build hadoop-2.7 with Hive 1.2.1, build hadoop-3.2 with Hive 2.3.5.`?

Reply:
I think we can just say, `By default, Spark will build hadoop-2.7 with Hive 1.2.1, and can build with hadoop-3.2 and Hive 2.3.5 optionally.`